app/controllers/setup_controller.rb

```diff
@@ -19,9 +19,10 @@ def index
       return
     end
 
+    password = @user.notify_welcome(password)
```

Comment:
The variable `password` is undefined before this line, so I don't see how this will work?